docs/commands/lint.md

```diff
@@ -2,239 +2,378 @@
+| --format | string | [Applies formatting to the output](#how-to-format-linting-results). Possible values: `codeframe`, `stylish`, `json`, `checkstyle`. Default: `codeframe`. |
+| --generate-ignore-file | boolean | [Generates an ignore file](#how-to-ignore-specific-errors-and-warnings) which tells Redocly CLI to ignore certain warnings/errors when validating the definition. |
+| --help | boolean | [Displays a short help menu](#how-to-open-the-help), showing lint options. |
+| --max-problems | integer | [Truncates the output](#how-to-limit-issues-shown-in-the-output) to show only a specified number of errors and warnings. Default: `100`. |
+| --skip-preprocessor | array | [Ignores specific preprocessors](#how-to-skip-a-rule-or-preprocessor-during-linting). |
+| --skip_rule | array | [Ignores specific custom or built-in rules](#how-to-skip-a-rule-or-preprocessor-during-linting). |
+| --version | boolean | Adding `--version` to any command (not just `lint`) will show your current Redocly CLI version. |
 
-### Entrypoints
+## Usage examples
 
-The `lint` command behaves differently depending on how you pass entrypoints to it and whether the [configuration file](#custom-configuration-file) exists.
+### How to use entrypoints to check all or selected definitions
 
-#### Pass entrypoints directly
+##### Method 1: Pass entrypoints directly
+In this case, `lint` will validate the definitions passed in the command and ignore the `apis` section in `redocly.yaml` or any other customized config file you may be using.
 
 ```bash
-openapi lint openapi/openapi.yaml
+redocly lint openapi/openapi.yaml
 ```
 
-In this case, `lint` will validate the definition(s) passed to the command. The configuration file is ignored.
+The `entrypoints` argument can also use any glob format supported by your file system. For example, `redocly lint ./root-documents/*.yaml`.
 
-The `entrypoints` argument can also use any glob format supported by your file system. For example, `openapi lint ./root-documents/*.yaml`.
+##### Method 2: Pass entrypoints via the config file
 
-#### Pass entrypoints via configuration file
+The `apis` section in the Redocly configuration file (`redocly.yaml`) lets you configure one or more APIs. Each file path is specified (using the `root` property), but files are identified by their name and version in the format `name@version`.
 
-Instead of full paths, you can use names listed in the `apis` section of your Redocly configuration file as entrypoints.
+`@version` is optional, and when not provided, Redocly apps interpret it as `latest` by default. Every `name@version` combination listed in the `apis` section must be unique.
 
-```bash Command
-openapi lint main
-```
+You can pass `name` or `name@version` in commands like `lint` and `stats` to act only on those APIs. Below is an example `apis` section showing three files identified using `main`, `production` and `sandbox`:
 
 ```yaml Configuration file
```

Comment:
This file goes beyond a minimal viable example. It is also incorrect everywhere:

Wrong
```
lint: []
```

Correct
```
lint: {}
```

One is an array, and one is an object. The same issue with `features.openapi`.

Those shouldn't be fixed though. They should be removed here because it is a configuration of nothing. The minimum way to do the same thing is to remove the lines, making the file shorter and easier to digest.

Reply:
Sorry Adam. I don't know what you mean or what action I should take here.